internal/controller/operatorconfigmap_controller.go

```diff
@@ -1075,3 +1089,74 @@ func parseTolerations(csiPluginTolerations string) ([]corev1.Toleration, error)
 	}
 	return tolerations, nil
 }
+
+func (c *OperatorConfigMapReconciler) removeNoobaa() error {
+	noobaa := &nbv1.NooBaa{}
+	noobaa.Name = "remote-noobaa"
+	noobaa.Namespace = c.OperatorNamespace
+
+	if err := c.get(noobaa); client.IgnoreNotFound(err) != nil {
```

Comment:
If the noobaa CRD is not on the cluster, this get operation will return a different value that should also be ignored